.github/workflows/gradle.yml

```diff
@@ -41,16 +41,17 @@ jobs:
       build: ${{ steps.filter.outputs.build }}
       cdk: ${{ steps.filter.outputs.cdk }}
       cli: ${{ steps.filter.outputs.cli }}
-      connectors: ${{ steps.filter.outputs.connectors }}
+      connectors_base: ${{ steps.filter.outputs.connectors_base }}
       db: ${{ steps.filter.outputs.db }}
-      frontend: ${{ steps.filter.outputs.frontend }}
+      any_change: ${{ steps.filter.outputs.any_change }}
     steps:
       - name: Checkout Airbyte
         uses: actions/checkout@v3
       - uses: dorny/paths-filter@v2
         id: filter
         with:
           # Note, the following glob expression within a filters are ORs.
+          # Note, if no filters match, the steps are all skipped. This is why we have the any_change filter.
```

Comment:
As i understand it, the jobs are skipped before even checking the filters, which is the problem? 

Now, we'll strat the `start-connectors-base-build-runner` job, the `if` will eval to false so not run the steps, and then the required check for the job still returns green? 

if that's the case, maybe some more clarity in this comment will be helpful

Reply:
Good call out. 

Looking at the output of the failed build it was essentially because the action overwrote the `modified_files` variable used by GHA. Which caused them to be skipped without being partially ran. 

Which as I understand had them not reporting a status check back to github

will update that comment